<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>client::ManageTrackFindONVIFDeviceWindow</class>
 <widget class="QDialog" name="client::ManageTrackFindONVIFDeviceWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>923</width>
    <height>641</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add ONVIF Source</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_10">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout_6">
     <item row="0" column="0">
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Discovery</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_4" columnstretch="0,0,0">
           <item row="3" column="0" colspan="3">
            <widget class="QPushButton" name="buttondiscoverrefresh">
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QCheckBox" name="checkipv6">
             <property name="text">
              <string>IPv6</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QCheckBox" name="checklinklocal">
             <property name="text">
              <string>Link Local</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QCheckBox" name="checkipv4">
             <property name="text">
              <string>IPv4</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="3">
            <widget class="ManageTrackFindONVIFDeviceDiscoveryTree" name="treediscovery">
             <property name="minimumSize">
              <size>
               <width>320</width>
               <height>0</height>
              </size>
             </property>
             <column>
              <property name="text">
               <string>Name</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Location</string>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Address</string>
              </property>
             </column>
            </widget>
           </item>
           <item row="1" column="1" colspan="2">
            <widget class="QLineEdit" name="edittextfilter"/>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Text Filter:</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="gridLayoutWidget">
        <layout class="QGridLayout" name="gridLayout">
         <item row="1" column="0">
          <widget class="QGroupBox" name="groupBox_4">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="minimumSize">
            <size>
             <width>360</width>
             <height>0</height>
            </size>
           </property>
           <property name="title">
            <string>Test</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_11">
            <item row="0" column="0">
             <widget class="QScrollArea" name="scrollArea">
              <property name="frameShape">
               <enum>QFrame::NoFrame</enum>
              </property>
              <property name="widgetResizable">
               <bool>true</bool>
              </property>
              <widget class="QWidget" name="scrollAreaWidgetContents">
               <property name="geometry">
                <rect>
                 <x>0</x>
                 <y>0</y>
                 <width>440</width>
                 <height>413</height>
                </rect>
               </property>
               <layout class="QGridLayout" name="gridLayout_12">
                <item row="0" column="0">
                 <layout class="QGridLayout" name="gridLayout_7">
                  <item row="0" column="0" rowspan="2">
                   <widget class="QLabel" name="labeltestoutput">
                    <property name="text">
                     <string/>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QPushButton" name="buttontest">
              <property name="text">
               <string>Test</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QGroupBox" name="groupBox">
           <property name="title">
            <string>Source</string>
           </property>
           <layout class="QGridLayout" name="gridLayout_3">
            <item row="2" column="0">
             <layout class="QGridLayout" name="gridLayout_2">
              <item row="0" column="0">
               <widget class="QLabel" name="label_4">
                <property name="text">
                 <string>URI:</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0" rowspan="5">
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Username:</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="edituri">
                <property name="toolTip">
                 <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Examples: rtsp://1.2.3.4/media.sdp&lt;/p&gt;&lt;p&gt;http://2.3.4.5/onvif/device_service&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
                </property>
               </widget>
              </item>
              <item row="1" column="1" rowspan="5">
               <widget class="QLineEdit" name="editusername"/>
              </item>
              <item row="6" column="1" rowspan="5">
               <widget class="QLineEdit" name="editpassword">
                <property name="echoMode">
                 <enum>QLineEdit::Password</enum>
                </property>
               </widget>
              </item>
              <item row="6" column="0" rowspan="5">
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Password:</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <layout class="QGridLayout" name="gridLayout_9">
           <item row="0" column="1">
            <widget class="QPushButton" name="buttoncancel">
             <property name="text">
              <string>Cancel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QPushButton" name="buttonok">
             <property name="text">
              <string>OK</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ManageTrackFindONVIFDeviceDiscoveryTree</class>
   <extends>QTreeWidget</extends>
   <header>monocleclient/managetrackfindonvifdevicediscoverytree.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>treediscovery</tabstop>
  <tabstop>edittextfilter</tabstop>
  <tabstop>checkipv4</tabstop>
  <tabstop>checkipv6</tabstop>
  <tabstop>checklinklocal</tabstop>
  <tabstop>buttondiscoverrefresh</tabstop>
  <tabstop>edituri</tabstop>
  <tabstop>editusername</tabstop>
  <tabstop>editpassword</tabstop>
  <tabstop>scrollArea</tabstop>
  <tabstop>buttonok</tabstop>
  <tabstop>buttoncancel</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
